scylla-cql/src/frame/value_tests.rs

```diff
@@ -124,6 +124,32 @@ fn bigint03_serialization() {
     }
 }
 
+#[cfg(feature = "num-bigint-04")]
+#[test]
+fn bigint04_serialization() {
+    let cases_from_the_spec: &[(i64, Vec<u8>)] = &[
+        (0, vec![0x00]),
+        (1, vec![0x01]),
+        (127, vec![0x7F]),
+        (128, vec![0x00, 0x80]),
+        (129, vec![0x00, 0x81]),
+        (-1, vec![0xFF]),
+        (-128, vec![0x80]),
+        (-129, vec![0xFF, 0x7F]),
+    ];
```

Comment:
This is the fourth test case that depends on those cases and the fourth time they appear in the code. Please deduplicate.